.gitignore

```diff
@@ -0,0 +1,29 @@
+# Compiled class file
```

Comment:
Ah, no google-cloudevents here... I think it would be best to *either* add it as a submodule, or have it in .gitignore.

Reply:
Added.